data/data.json

```diff
@@ -1482,6 +1482,72 @@
     "v": "2"
    }
   },
+  "v1.18": {
```

Comment:
Once https://github.com/rancher/kontainer-driver-metadata/pull/242 is merged to the 2.4 line, this generated data commit should be scrapped in favor of using `go generate` to pull the latest from the dev-v2.4 branch